src/App.tsx

```diff
@@ -1,3 +1,7 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 
-export default () => <div className="App">EAZY LEARN</div>;
+const App: React.FC = function () {
```

Comment:
I would prefer function declaration or using arrow function, I don't understand why you used function expression, you can try to explain me your choice

Reply:
Whoops, miss that, i was thinking i'm writing an arrow function